crates/sui-json-rpc/src/governance_api.rs

```diff
@@ -84,33 +86,49 @@ impl GovernanceReadApi {
             self.get_system_state()?.into_sui_system_state_summary();
         let mut delegated_stakes = vec![];
         for ((pool_id, validator_address), stakes) in pools {
+            // Rate table and rate can be null when the pool is not active
```

Comment:
Technically candidate staking pools have empty rate tables too. It's just that today `get_exchange_rate_table` doesn't support candidate staking pools.